source/common/redis/command_splitter_impl.cc

```diff
@@ -164,7 +164,9 @@ SplitRequestPtr InstanceImpl::makeRequest(const RespValue& request, SplitCallbac
     }
   }
 
-  auto handler = command_map_.find(request.asArray()[0].asString());
+  std::string to_lower_string(request.asArray()[0].asString());
+  to_lower_table_.toLowerCase(to_lower_string);
```

Comment:
thoughts on https://github.com/google/protobuf/blob/master/src/google/protobuf/stubs/strutil.h#L176 instead of the LUT approach?

may be useful other places and has the benefit of not requiring table instantiation in each case.